cmd/atlas/mux.go

```diff
@@ -60,3 +62,12 @@ func parseDSN(url string) (string, string, error) {
 	}
 	return a[0], a[1], nil
 }
+
+func schemaNameFromDSN(url string) (string, error) {
+	_, dsn, err := parseDSN(url)
+	if err != nil {
+		return "", err
+	}
+	a := strings.SplitAfter(dsn, "/")
+	return a[len(a)-1], nil
+}
```

Comment:
qq: why not use a proper parser here? Both MySQL and PostgreSQL drivers have diff. Let me know if you need help with this.

Reply:
https://pkg.go.dev/github.com/jackc/pgconn#Config

https://pkg.go.dev/github.com/go-sql-driver/mysql#Config

Reply:
👍